.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
```

Comment:
Some PRs are simple to a point of being self-explanatory, so perhaps reword this to become:

> If necessary, go into depth of what this pull-request is doing.